.travis.yml

```diff
@@ -297,33 +297,20 @@ script:
   - SKIP_FILES3="|shippable.yml|shippable.sh|wercker.yml|wercker.sh|intellij-idea-inspections.xml"
   - SKIP_FILES4="|Jenkinsfile"
   - SKIP_FILES=$SKIP_FILES1$SKIP_FILES2$SKIP_FILES3$SKIP_FILES4
-  - SKIP_CI_BY_COMMIT="false"
+  - source ./.ci/all.sh
   - |
-    if [[ $SKIP_CI_BY_FILES != 'false' ]]; then
-         if [[ $(git diff --name-only HEAD `git merge-base origin/master HEAD` \
-                    | grep -vE "$SKIP_FILES" | cat | wc -c | sed 's/^ *//' ) > 0 ]]; then
-              SKIP_CI_BY_FILES="false"
-         else
-              SKIP_CI_BY_FILES="true"
-         fi
-    fi
-  - |
-    export SKIP_CI_BY_COMMIT="false"
-    if [ $(git log -1 | grep -E "\[maven-release-plugin\] prepare release" | cat | wc -l) -lt 1 ];
-         then
-           SKIP_CI_BY_COMMIT="false"
-         else
-           SKIP_CI_BY_COMMIT="true"
+    if [ -z ${SKIP_CI_BY_FILES+x} ]; then
+        SKIP_CI_BY_FILES="recheck"
     fi
-  - echo "SKIP_CI_BY_FILES="$SKIP_CI_BY_FILES
-  - echo "SKIP_CI_BY_COMMIT="$SKIP_CI_BY_COMMIT
+  - should_run_ci $SKIP_CI_BY_FILES $SKIP_FILES true
   - |
     set -e
-    if [[ $SKIP_CI_BY_FILES == 'false' && $SKIP_CI_BY_COMMIT == 'false' ]]; then
+    if [[ $RUN_CI == 1 ]]; then
          if [[ $USE_MAVEN_REPO == 'true' && ! -d "~/.m2" ]]; then
            echo "Maven local repo cache is not found, initializing it ..."
            mvn -B install -Pno-validations;
          fi
+         echo "eval of CMD is starting";
```

Comment:
lets print CMD value here, it is very useful to see exact command  during investigation of CI failure
`echo CMD=$CMD`

Reply:
Done.